src/app/pages/rpc-send-transfer/use-rpc-send-transfer-actions.tsx

```diff
@@ -18,8 +18,14 @@ import { useRpcSendTransferContext } from './rpc-send-transfer.context';
 
 export function useRpcSendTransferActions() {
   const { availableBalance, currentEditorFee } = useFeeEditorContext();
-  const { amountAsMoney, isLoading, recipients, requestId, tabId, utxos } =
-    useRpcSendTransferContext();
+  const {
+    amount: amountAsMoney,
```

Comment:
Can this now just be `amount`?

Reply:
Yep, I'll change, was going that route but wasn't sure if people are ok with `amount.amount` bc of the Money type.